lib/src/action.dart

```diff
@@ -65,6 +67,7 @@ class Action<T> implements Function {
   /// Cancel all subscriptions that exist on this [Action] as a result of
   /// [listen] being called. Useful when tearing down a flux cycle in some
   /// module or unit test.
+  @Deprecated('Use (and await) dispose() instead.')
```

Comment:
Yeah, I don't mind that. I do like the describing how to fix the deprecation though, which w_transport doesn't seem to do.